sdk/include/opentelemetry/sdk/metrics/aggregation/histogram_aggregation.h

```diff
@@ -13,57 +13,47 @@ namespace sdk
 {
 namespace metrics
 {
-template <class T>
-static inline void PopulateHistogramDataPoint(HistogramPointData &histogram,
-                                              opentelemetry::common::SystemTimestamp epoch_nanos,
-                                              T sum,
-                                              uint64_t count,
-                                              std::vector<uint64_t> &counts,
-                                              std::vector<T> boundaries)
-{
-  histogram.epoch_nanos_ = epoch_nanos;
-  histogram.boundaries_  = boundaries;
-  histogram.sum_         = sum;
-  histogram.counts_      = counts;
-  histogram.count_       = count;
-}
 
 class LongHistogramAggregation : public Aggregation
```

Comment:
Could this be a template class as it almost the same as `DoubleHistogramAggregatoin` below?

Reply:
This was discussed in one of the earlier PR [here](https://github.com/open-telemetry/opentelemetry-cpp/pull/1178#discussion_r789262011):
--snip
Preferred overloaded methods over the template ( for double and long type) to avoid having header-only metrics sdk. The template use had a cascading effect here, forcing to make all inter-related classes as templates. Since this is not a customer-facing API, we have the flexibility to change it later.
--snap